src/blockChainManager/getLastBlock.ts

```diff
@@ -0,0 +1,4 @@
+import { block } from './block';
+
+const getLastBlock = (blockChain:block[]) => blockChain[blockChain.length];
```

Comment:
you should specify the return type =>  `getLastBlock = (blockChain:block[])  :  block => blockChain[blockChain.length];`